app/controllers/protected_areas_controller.rb

```diff
@@ -92,10 +96,25 @@ def load_other_designations
     other_designations.reject { |pa| pa.id == @protected_area.id }
   end
 
-  TRANSBOUNDARY_SITES = "Transboundary sites".freeze
-  def load_networks
-    networks = @protected_area.networks.reject(&:designation)
-    # ensure that transboundary sites network always appears first
-    networks.sort { |a,b| a.name == TRANSBOUNDARY_SITES ? -1 : a.name <=> b.name }
+  # TODO: Methods, models, controllers, modules related to Networks are slated for removal
+
+  # TRANSBOUNDARY_SITES = "Transboundary sites".freeze
+  # def load_networks
+  #   networks = @protected_area.networks.reject(&:designation)
+  #   # ensure that transboundary sites network always appears first
+  #   networks.sort { |a,b| a.name == TRANSBOUNDARY_SITES ? -1 : a.name <=> b.name }
+  # end
+
+  def get_other_sites
```

Comment:
@stanleypliu 2 points here:

1. I would use `if @protected_area.is_transboundary` as it already contains that kind of logic
2. I wonder if we should return other areas within the same country if not transboundary, rather than any other areas around the world? For example, it feels a bit odd that, if I'm looking at a PA in Italy, I see other 3 sites which are in a completely different country

Point 2 is perhaps more of a question for @stacytalbot

Reply:
Thanks @stacytalbot . My question was more about when looking at a site which is 'not' transboundary.
If 'not' transboundary, the query below gets any 3 sites which are not the one I'm looking at; so those sites could be anywhere.

It works well already if the site I'm looking at 'is' transboundary

Reply:
That's been done - should return sites in the same country now if not transboundary.